Controllers/err.c.js

```diff
@@ -1,8 +1,7 @@
-exports.handleInvalidPaths = (req, res, next) => {
+exports.handleInvalidPaths = (err, req, res, next) => {
```

Comment:
```js
exports.handleInvalidPaths = (err, req, res, next) => {
```
Adding the extra `err` parameter here will make this an error handler and stop it from picking up invalid paths. This is not what you want it to do and should put it back to only having 3 parameters